### DISCUSSION TITLE
Extend media library for MP4 videos

### DISCUSSION THREAD

**Discussion by afiller** [typemill/typemill#437] [Ideas]
At first: 
Thank you for typemill! It's exactly what I searched for. Uncomplicated, light-weight, customisable, perfect.

At second: 
I wanted to integrate a video in my website and thus I played around with the YouTube content format. It only showed a URL on my machine in the standard template, but it also lead me to the idea that the video could also be hosted locally without YouTube. Analog to images you could only use the <video> tag (https://www.w3schools.com/tags/att_video_src.asp) and the media library could support the upload of MP4 videos.

Best regards,

Andreas

**Comment by trendschau** [comment]
Hi Andreas, yes, that might be a nice feature, I will add it to the list and think about it. Right now you can only add youtube-videos, Typemill will only store a preview image with a link to the youtube video and load it after a user clicks on the play button.
